backend/Program.cs

```diff
@@ -14,6 +20,37 @@
     options.UseNpgsql(builder.Configuration.GetConnectionString("DefaultConnection"));
 });
 
+builder.Services.AddIdentity<User, IdentityRole>(options => {
+    options.Password.RequireDigit = false;
+    options.Password.RequireLowercase = false;
+    options.Password.RequireUppercase = false;
+    options.Password.RequireNonAlphanumeric = false;
+    options.Password.RequiredLength = 6;

```

Comment:
Can be removed since 6 is the default value

Reply:
True, thanks for the heads up! These should also be considered changing later on.